content/scalers/postgresql.md

```diff
@@ -0,0 +1,91 @@
++++
+fragment = "content"
+weight = 100
+title = "PostgreSQL"
+background = "light"
++++
+
+Scale applications based on a PostgreSQL query
+
+**Availability:** v1.2+ | **Maintainer:** Community
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `postgresql` trigger that scales based on a postgresql query
+
+The Postgresql scaler allows for two connection options:
+
+A user can offer a full connection string 
+(often in the form of an environment variable secret)
+
+- `connectionString` MySQL connection string that should point to environment variable with valid value
+
+Alternatively, a user can specify individual
+arguments (host, userName, password, etc.), and the scaler will form a connection string 
+internally.
+- `host:` Service URL to postgresql. Note that you should use a full svc URL as KEDA will need to contact postgresql from a different namespace
+- `userName:` Username for postgresql user
+- `password:` Password for postgresql user
+- `port:` Postgresql port
+- `dbName:` Postgresql Database name
+- `sslmode:` SSL policy for communicating with database
+
+Finally, a user inserts a query that returns the desired value
+
+- `query:` What query to poll postgresql with. Query must return an integer.
+
+This is an example of using a full connection string:
+```yaml
+  triggers:
+    - type: postgresql
+      metadata:
+        connStr: AIRFLOW_CONN_AIRFLOW_DB
+        query: "SELECT ceil(COUNT(*)::decimal / 16) FROM task_instance WHERE state='running' OR state='queued'"
+```
+
+While this is an example of specifying each parameter:
+
+```yaml
+  triggers:
+    - type: postgresql
+      metadata:
+        userName: "kedaUser"
+        password: PG_PASSWORD
+        host: postgres-svc.namespace.cluster.local #use the cluster-wide namespace as KEDA 
+                                                   #lives in a different namespace from your postgres
+        port: "5432"
+        dbName: postgresql
+        sslmode: disable
+        query: "SELECT ceil(COUNT(*)::decimal / 16) FROM task_instance WHERE state='running' OR state='queued'"
+```
+
+### Authentication Parameters
+
+You can authenticate by using a password or store the password within the connStr.
```

Comment:
Does this mean we don't support trigger auth for connection string or is it just not covered here?

Reply:
@tomkerkhove I've added the connection string to the trigger auth